Tavisca.Bootcamp.LanguageBasics.Exercise2/Program.cs

```diff
@@ -13,18 +14,64 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string[] postTimes, string[] showTimes, string expected)
-        {
+        private static void Test(string[] postTimes, string[] showTimes, string expected)
+        {
             var result = GetCurrentTime(postTimes, showTimes).Equals(expected) ? "PASS" : "FAIL";
             var postTimesCsv = string.Join(", ", postTimes);
             var showTimesCsv = string.Join(", ", showTimes);
-            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
+            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
         }
 
-        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
+            int i=0;
+            int len=exactPostTime.Length;
+            String[] arr = new String[len];
+            int k=0;
+            for (int z = 0; z < exactPostTime.Length; z++)
+            {
+                for (int j = z + 1; j < exactPostTime.Length; j++)
+                {
+                    if (exactPostTime[z] == exactPostTime[j])
+                    {
+                        if (showPostTime[z] != showPostTime[j])
+                        {
+                            return "impossible";
+                        }
+                    }
+                }
+            }
+           foreach(string s in exactPostTime)
+           {
+                int hr = Int32.Parse(s.Substring(0,2));
+                int min =Int32.Parse(s.Substring(3,2));
+                int sec = Int32.Parse(s.Substring(6,2));
+                DateTime d = new DateTime(2019, 1, 1, hr, min, sec);
+               if(showPostTime[i].Contains("seconds"))
+               {
+                    arr[i]=s;
+               }
+               else if(showPostTime[i].Contains("minutes"))
+                {
+                   int temp = Int32.Parse(showPostTime[i].Substring(0,2));
+                   d=d.AddMinutes(temp);
+                   arr[i]=d.ToLongTimeString();
+                   
+                }
+                else if(showPostTime[i].Contains("hours"))
+                {
+                   int t = Int32.Parse(showPostTime[i].Substring(0,2));
+                   d=d.AddHours(t);
+                   arr[i]=d.ToLongTimeString();
+                }
+               i++;
+               k++;
+           }
+           Array.Sort(arr);
+            String res = arr[len-1].Split(" ")[0];

```

Comment:
Please **don't use the meaningless variable name**. **Avoid using A,b,c,d etc** and also don't use abbreviations for eg: **_res should be result_**